### ISSUE TITLE
Undefined `cslreferences` with updated pandoc

### ISSUE THREAD

**Issue by dan-fritchman** [miki725/md2pdf-ieee-sample#1]
Running the `bibliography` example generates this error:

```
bibliography $ pandoc \
    --from=markdown \
    --to=latex \
    --template=template.latex \
    --filter=pandoc-crossref \
    --filter=pandoc-citeproc \
    --bibliography=bibliography.bib \
    --csl=bibliography.csl \
    --output=bibliography.pdf \
    bibliography.md
Error producing PDF.
! LaTeX Error: Environment cslreferences undefined.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.131 \begin{cslreferences}
```

**Comment by dan-fritchman** [comment]
Just requires adding a template-section along these lines:

```
$if(csl-refs)$
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newenvironment{cslreferences}%
  {$if(csl-hanging-indent)$\setlength{\parindent}{0pt}%
  \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces$endif$}%
  {\par}
$endif$
```

Included in PR #2

**Comment by miki725** [comment]
hmm wonder why it worked previously for me for https://miki725.com/2019/10/15/markdown-to-pdf-ieee.html#bibliography

thanks for the fix!
